processing/src/main/java/org/apache/druid/segment/incremental/OnheapIncrementalIndex.java

```diff
@@ -89,10 +93,11 @@
       boolean concurrentEventAdd,
       boolean sortFacts,
       int maxRowCount,
-      long maxBytesInMemory
+      long maxBytesInMemory,
+      boolean preserveExistingMetrics
```

Comment:
same comment about javadoc and comments about reingestion

Reply:
Done